### ISSUE TITLE
subscribed rules renew failed,rules will not appiled

### ISSUE THREAD

**Issue by yhlnac** [salarcode/SmartProxy#305]
### Description

When I upgraded to version 1.2，I have subscribed gfwlist.txt
After restarting the computer, the subscription will fail, and then the rules of the subscription will not be applied, just like disappearing

### Which browsers

Other (Chromium based)

### Browser versions

Chromium 102.0.5005.167

### Screenshots

<img width="991" alt="111" src="https://github.com/salarcode/SmartProxy/assets/97741385/9e0fa74c-eb18-455d-a77d-9cb9a3ff98d9">

**Comment by LightAPIs** [comment]
### Environment

Chrome v114.0.5735.199 && SmartProxy v1.2.1

### Reason

Found that the `reloadEmptyRulesSubscriptions()` method fires every time the extension is reloaded. At this time, the content in `proxyRules` is actually empty(*Even if the last load had been successful*):

![console](https://github.com/salarcode/SmartProxy/assets/47911535/4a34c44a-47de-464b-96d9-a66b63fa57f6)

So when the network itself cannot access `githubusercontent.com`, it's not possible to obtain subscription rules normally.

![network](https://github.com/salarcode/SmartProxy/assets/47911535/b45db774-de76-40ff-a0eb-efe93ae3d244)

### Solution

The solution on the client side: add a local rule for `githubusercontent.com`.

![rule](https://github.com/salarcode/SmartProxy/assets/47911535/f0615e40-427d-42d9-9e0c-9d651f9d5db1)

**Comment by salarcode** [comment]
Thanks for your discovery, so `githubusercontent.com` is blocked by your ISP!
This will be the case when active profile is `Smart Proxy`, on `Always Enabled` profile the subscriptions should work.

Your solution of just adding `githubusercontent.com` to personal rules is elegant

The subscription rules being empty on startup will be resolved by #299

**Comment by salarcode** [comment]
Closing as duplicate of #299
